.gitignore

```diff
@@ -70,3 +69,7 @@ CreeDictionary/CreeDictionary/static/CreeDictionary/js/index.js
 CreeDictionary/CreeDictionary/static/CreeDictionary/css/styles.css
 *.css.map
 *.js.map
+
+# .dockerignore
+# see dockerignore-example.txt if you want to publish an image
+.dockerignore
```

Comment:
instead of tar-ing, using a combination of `COPY` and `.dockerignore` with root directory build context now. The speed is not that slow. The reason for the switching is that this way, local files that should be ignored while publishing the package can be easily specified. I have a lot of BIG local files in my work tree, including cache, database, database backups, stuff. It takes LONGER time to tar them.